src/mme/s1ap-handler.c

```diff
@@ -2462,58 +2527,24 @@ void s1ap_handle_handover_required(mme_enb_t *enb, ogs_s1ap_message_t *message)
         return;
     }
 
-    switch (TargetID->present) {
-    case S1AP_TargetID_PR_targeteNB_ID:
-        ogs_s1ap_ENB_ID_to_uint32(
-            &TargetID->choice.targeteNB_ID->global_ENB_ID.eNB_ID,
-            &target_enb_id);
+    switch (*HandoverType) {
+    case S1AP_HandoverType_intralte:
+        s1ap_handle_handover_required_intralte(source_ue, Cause, TargetID, Source_ToTarget_TransparentContainer);
         break;
-    default:
-        ogs_error("Not implemented(%d)", TargetID->present);
+    case S1AP_HandoverType_ltetoutran:
+    case S1AP_HandoverType_ltetogeran:
+    case S1AP_HandoverType_utrantolte:
+    case S1AP_HandoverType_gerantolte:
+    case S1AP_HandoverType_eps_to_5gs:
```

Comment:
@acetcom I assumed S1AP_HandoverType_eps_to_5gs and S1AP_HandoverType_fivegs_to_eps, hope that's correct.